site/src/App/DocNavigation/DocSnippets.tsx

```diff
@@ -21,7 +20,7 @@ export const DocSnippets = () => {
           snippets.map(({ group, name, code }) => (
             <Stack space="medium" key={`${group}_${name}`}>
               <Text tone="secondary">{name}</Text>
-              <BraidProvider styleBody={false} theme={theme}>
+              <BraidProvider styleBody={false} theme={docsTheme}>
```

Comment:
Can we remove this provider all together? If `DocExample` is managing the selected theme, this provider may be unnecessary.